File: DataEntry/ClientApp/src/types/listing/parkings.ts
export interface Parkings {
    ratio: number | null,
    ratioPer: number | null,
    ratioPerUnit: string,
    parkingDetails: ParkingDetails[]
}
 
export interface ParkingDetails {
    parkingType: string,
    parkingSpace: number | null,
    amount: number | null,
    interval: string,
    currencyCode: string
}

File: DataEntry/Data/DBContext/Model/Parkings.cs
using System.Collections.Generic;
namespace dataentry.Data.DBContext.Model
{
    public class Parkings
    {
        public double? Ratio { get; set; }
        public double? RatioPer { get; set; }
        public string RatioPerUnit { get; set; }
        public List<ParkingDetail> ParkingDetails { get; set; } 
    }
}
